libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -497,6 +499,21 @@ void AP_Camera::take_picture()
     GCS_MAVLINK::send_to_components(MAVLINK_MSG_ID_COMMAND_LONG, (char*)&cmd_msg, sizeof(cmd_msg));
 }
 
+void AP_Camera::prep_mavlink_msg_camera_feedback(uint64_t timestamp_us)
+{
+    const AP_AHRS &ahrs = AP::ahrs();
+    if (!ahrs.get_position(feedback.location)) {
+        // completely ignore this failure!  AHRS will provide its best guess.
+    }
+    feedback.timestamp_us = timestamp_us;
+    feedback.roll_sensor = ahrs.roll_sensor;
```

Comment:
Why not just use a vector 3f with a comment saying RPY?

Reply:
> Why not just use a vector 3f with a comment saying RPY?

These are integers, not floats.... and we seldom use vector3i for these sorts of things.  I did consider using a Vector but decided against it.  Naming was the chief thing there; `Vector<int32_t> attitude_sensor` is just nasty, but you kind of need `_sensor` in there is that's how we kind of mark things as being cdeg.